std/experimental/normint.d

```diff
@@ -0,0 +1,571 @@
+// Written in the D programming language.
+
+/**
+This module implements support for normalized integers.
+
+Authors:    Manu Evans
+Copyright:  Copyright (c) 2015, Manu Evans.
+License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
+Source:     $(PHOBOSSRC std/experimental/normint.d)
+*/
+module std.experimental.normint;
```

Comment:
I don't really like `normint` as a module name. Maybe we could put this into the same package as checkedint. Is there a good category term for normalized integers, checked integers (maybe bigint) and similar stuff? `std.integer.normalized, std.integer.checked/bounded std.integer.bigint`?